definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -608,6 +608,13 @@
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "CRUD"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "D8ConfidentialDocumentsUploaded",
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.